packages/babel-plugin/src/utils/state-manager.js

```diff
@@ -163,6 +164,14 @@ export default class StateManager {
       'options.debug',
     );
 
+    const enableDebugClassNames: StyleXStateOptions['enableDebugClassNames'] =
+      z.logAndDefault(
+        z.boolean(),
+        options.enableDebugClassNames ?? true,
+        true,
+        'options.enableDebugClassNames',
+      );
+
```

Comment:
This option should default to the value of `debug`.

Also, should we call this: `debugClassPrefix` maybe?

```suggestion
    const enableDebugClassNames: StyleXStateOptions['enableDebugClassNames'] =
      z.logAndDefault(
        z.boolean(),
        options.enableDebugClassNames ?? debug,
        true,
        'options.enableDebugClassNames',
      );

```

Reply:
previously had this approach re: Nicolas' comment above